sdk/keyvault/Azure.Security.KeyVault.Secrets/api/Azure.Security.KeyVault.Secrets.netstandard2.0.cs

```diff
@@ -97,6 +97,7 @@ public partial class SecretClientOptions : Azure.Core.ClientOptions
     {
         public SecretClientOptions(Azure.Security.KeyVault.Secrets.SecretClientOptions.ServiceVersion version = Azure.Security.KeyVault.Secrets.SecretClientOptions.ServiceVersion.V7_6_Preview_1) { }
         public bool DisableChallengeResourceVerification { get { throw null; } set { } }
+        public Microsoft.Extensions.Logging.ILoggerFactory LoggerFactory { get { throw null; } set { } }
```

Comment:
This depends on the long-term plan for ClientOptions / how it will work with ClientPipelineOptions